### PR TITLE
fix(sdk/dsl) Made ContainerOp.output_artifact_paths private

### PR DESCRIPTION
**Pull request by Ark-kun**
It wasn't supposed to be public.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1832)

### CODE REVIEW

File: sdk/python/kfp/dsl/_container_op.py [hunk 1]
```diff
@@ -1023,7 +1023,7 @@ def _decorated(*args, **kwargs):
 
         # attributes specific to `ContainerOp`
         self.file_outputs = file_outputs
-        self.output_artifact_paths = output_artifact_paths or {}
+        self._output_artifact_paths = output_artifact_paths or {}
```
Comment (gaoning777) [inline sdk/python/kfp/dsl/_container_op.py]: Are there any functions that update the output_artifact_paths? If not, maybe we should as we make it private. WDYT?

Comment (Ark-kun) [reply sdk/python/kfp/dsl/_container_op.py]: > Are there any functions that update the output_artifact_paths? If not, maybe we should as we make it private. WDYT?

I think this PR does exactly that. Did you mean `file_outputs`?

**Comment by DavidSpek** [comment]
@Ark-kun I think this needs a rebase.

**Pull request closed without merging**